config/muxcable.py

```diff
@@ -730,13 +731,16 @@ def download(fwfile, port):
     if port is not None and port != "all":
 
         physical_port_list = []
-        get_physical_port_list(port, physical_port_list)
+        physical_port_list, asic_index = get_physical_port_list(port)
+        f = open("demofile2.txt", "w")
+        f.write("yes ")
+        f.write("{} ".format(len(physical_port_list)))
```

Comment:
Why write to this file?

Reply:
these were left-over debug statements, removed now